pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -162,6 +162,13 @@ type ScheduledSparkApplicationList struct {
 	Items           []ScheduledSparkApplication `json:"items,omitempty"`
 }
 
+// Specific SparkUI config parameters
+type UIConfig struct {
+	IngressClass *string `json:"ingressClass,omitempty"`
```

Comment:
What does it mean for `IngressClass` to be `nil`?

Reply:
As any json item is omitempty, its covering the case where UIconfig is defined but not IngressClass